src/tracer.js

```diff
@@ -46,24 +54,27 @@ const session = axios.create({
     httpsAgent: new https.Agent({ keepAlive: true }),
 });
 
-
 /**
  * Adds an event to the tracer
  * @param {proto.event_pb.Event} event The event to add
  * @param {Promise} [promise] A promise that resolves when the event handling is Done, if required.
+ * @param {Object} tracer Optional tracer
  */
 module.exports.addEvent = function addEvent(event, promise) {
+    const tracerObj = module.exports.getTrace();
+    if (!tracerObj) return;
```

Comment:
Use {}  we dont charge you by the character

Reply:
😄